presto-main/src/test/java/com/facebook/presto/block/TestBlockBuilder.java

```diff
@@ -114,9 +115,12 @@ public void testNewBlockBuilderLike()
         assertEquals(newArrayBlockBuilder.getRetainedSizeInBytes(), 5848);
     }
 
-    @Test
+    @Test(description = "T83877176 - Skipped because it OOMs")
```

Comment:
Task is internal to Facebook. I would file an GitHub issue and refer it here.

Reply:
Do we just want to add comment to `@Test` annotation  and have a comment to explain why it's commented out?

Reply:
Good point, I did open a GH issue for this. I'll fix it.
The reason will appear in the skip message so that should be enough.

Reply:
Fixed.